www/js/modules/main/directives/histogramChart.js

```diff
@@ -131,13 +131,13 @@ export default ($log) => {
         controller: histogramController,
         controllerAs: 'histogramController',
         bindToController: {
-            dataSet: '@'
+            dataset: '='
         },
         scope: {},
         restrict: 'E',
-        link: (scope, elm, attrs) => {
+        link: (scope, elm, attrs, ctrl) => {
             log.debug('histogramChartDirective.js - in link!');
-            let data = scope.dataSet;//histogramController().getData();
```

Comment:
Because we're using `bindToController` in this directive, the bound data is not available on the scope, but instead the controller. luckily, the controller is the fourth parameter to the link function,